src/field/field255.rs

```diff
@@ -515,23 +499,19 @@ mod tests {
         );
         assert_eq!(
             format!("{}", Field255::one()),
-            "0x0000000000000000000000000000000000000000000000000000000000000001"
+            "0x0100000000000000000000000000000000000000000000000000000000000000"
         );
         assert_eq!(
             format!("{}", -Field255::one()),
-            "0x7fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffec"
+            "0xecffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff7f"
         );
         assert_eq!(
             format!("{:?}", Field255::zero()),
             "0x0000000000000000000000000000000000000000000000000000000000000000"
         );
         assert_eq!(
             format!("{:?}", Field255::one()),
-            "0x0000000000000000000000000000000000000000000000000000000000000001"
-        );
-        assert_eq!(
-            format!("{:?}", -Field255::one()),
-            "0x7fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffec"
```

Comment:
It looks like this was redundant (line 522 in the previous version of the file)

Reply:
Yeah, probably a holdover from when the Debug and Display implementations differed. LGTM.